core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -290,6 +291,7 @@ class ReplicaManager(val config: KafkaConfig,
                      ) extends Logging {
   private val metricsGroup = new KafkaMetricsGroup(this.getClass)
   private val addPartitionsToTxnConfig = new AddPartitionsToTxnConfig(config)
+  val delayedShareFetchTimer = new SystemTimer("ShareFetch")
```

Comment:
The other thing I was thinking was to have `private val shareFetchPurgatooryName = "ShareFetch"` and use the val at both places so we in future do not change the names. Wdyt?